app/views/items/index.html.erb

```diff
@@ -104,7 +104,7 @@
                     <% elsif item.reserved? %>
                       <h3>This item is reserved.</h3>
                     <% else %>
-                      <h3><%= truncate(item.description, :length => 93) %></h3>
+                      <h3><%= truncate(item.description, :length => 85) %></h3>
```

Comment:
Also reduced the limit on this since it would expand the card too much, only reduced it by 8 chars, but this will keep all descriptions within 3 lines so the cards look better.